package-mac.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+mv TEMP-Grabber.app/Contents/MacOS/TEMP-Grabber TEMP-Grabber.app/Contents/MacOS/Grabber
+mkdir -p grabber-release
+ln -s  /Applications grabber-release/Applications 
+mv TEMP-Grabber.app grabber-release/Grabber.app
+macdeployqt grabber-release/Grabber.app
+hdiutil create -volname grabber -srcfolder grabber-release/ -ov -format UDRW -o grabber
```

Comment:
Is there any reason to use `macdeployqt` then `hdiutil` to create the DMG file, instead of directly using `macdeployqt -dmg`?